web/src/components/Wrapper.js

```diff
@@ -17,7 +20,13 @@ class Wrapper extends Component {
 
   render() {
     const { children } = this.props;
-    return <div className="site">{children}</div>;
+    let showSiteTitle = location.pathname === '/'
+    return (
+      <div className="site">
+        <Header key={location.pathname} showSiteTitle={showSiteTitle} />
```

Comment:
I'm not sure this should go in `Wrapper`, but 1) I could avoid using `withRoute` in the `Header` component this way, 2) I wasn't sure if there were another globally used component.